evm/chains/tester/__init__.py

```diff
@@ -19,33 +19,46 @@
     generate_vms_by_range,
 )
 
+from evm.rlp.headers import (
+    BlockHeader
+)
+
+from typing import (
```

Comment:
This should be at the top of the file together with other stdlib imports

Reply:
Alright, will move all of them to the top